sync.py

```diff
@@ -156,9 +165,6 @@ def delete_and_clear_cohort_details(self, concentration):
         return True
 
     def process_block(self, changed_banner_data, block_id, all_program_codes, time_to_wait):
-        if len(changed_banner_data) == 0:
-            return 'No data has been updated in Banner since the last sync; skipping sync of block ID "%s"' % block_id
-
```

Comment:
Why are you removing this check?

Reply:
Great question. I just realized it should be added back. It doesn't apply to the sync all, since we do this same check outside of this method. However, it needs to still be here for the sync id/path.

Added it back